Modules/SubModule_Import/xASL_imp_NII2BIDS_SubjectSessionRun.m

```diff
@@ -220,7 +237,9 @@
                 end
             end
             % Prioritize the Phoenix field values in the general case
-            jsonLocal.(fn{1}) = jsonDicom.PhoenixAnalyzed.(fn{1});
+            if ~(strcmp(fn{1},'EchoTime') && isHadamardFME)
```

Comment:
@jan-petr 
Does this change not break other cases? (i.e. Phoenix without `HadamardFME`)